### PR TITLE
Add .Phony to Makefile

### PR DESCRIPTION
**Pull request by razo7**
Similar to [NMO's PR](https://github.com/medik8s/node-maintenance-operator/pull/12), and [NHC's PR](https://github.com/medik8s/node-healthcheck-operator/pull/210).

Add [.PHONY](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) to each target.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -260,13 +276,12 @@ bundle-community-k8s: bundle-community ## Generate bundle manifests and metadata
...
 .PHONY: bundle-community
-bundle-community: bundle
-	#Add Community Edition suffix to operator name
+bundle-community: bundle ##Add Community Edition suffix to operator name
 	sed -r -i "s|displayName: Self Node Remediation Operator.*|displayName: Self Node Remediation Operator - Community Edition|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
 
 .PHONY: bundle-update
-bundle-update:
-    # update container image in the metadata
+bundle-update: ## Update containerImage, createdAt, skipRange, and icon fields in the bundle's CSV, then validate the bundle directory
```
Comment (mshitrit) [inline Makefile:283]: I think the comment is wrong, IIUC it only updates  containerImage & createdAt,

**Comment by mshitrit** [comment]
Just one NIT feel free to fix/unhold
/lgtm
/hold

File: Makefile (hunk 1 above)
Comment (razo7) [reply Makefile:283]: In [lines](https://github.com/medik8s/self-node-remediation/pull/122/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R288-R291) it does update  skipRange, and icon fields.

**Pull request merged**